exercises/list-ops/metadata.toml

```
@@ -1,2 +1,4 @@
 title = "List Ops"
 blurb = "Implement basic list operations."
+deep_dive_youtube_id = "WMpPO4WnKxg"
```

```suggestion
deep_dive_youtube_id = "3ppX9InEEAY"
```

YouTube broke. Reuploading. *sigh*